graylog2-web-interface/src/views/components/widgets/MessageTable.jsx

```diff
@@ -0,0 +1,216 @@
+// @flow strict
+import React from 'react';
+import * as Immutable from 'immutable';
+import styled from 'styled-components';
+
+import { AdditionalContext } from 'views/logic/ActionContext';
+import MessageFieldsFilter from 'logic/message/MessageFieldsFilter';
+import FieldType from 'views/logic/fieldtypes/FieldType';
+import MessagesWidgetConfig from 'views/logic/widgets/MessagesWidgetConfig';
+
+import { RefreshActions } from 'views/stores/RefreshStore';
+
+import { MessageTableEntry } from 'views/components/messagelist';
+import Field from 'views/components/Field';
+
+const Table = styled.table`
+  position: relative;
+  font-size: 11px;
+  margin-top: 15px;
+  margin-bottom: 60px;
+  border-collapse: collapse;
+  padding-left: 13px;
+  width: 100%;
+  word-break: break-all;
+
+  table.messages td, table.messages th {
+    position: relative;
+    left: 13px;
+  }
+
```

Comment:
All these styles are part of the `graylog2.less` file and children of the selector `.messages`. The next step is to implement the `tr` styles to the MassageTableEntry component as styled components. I don't want to blow up this PR even more, but will gladly do these changes afterwards.